### PR TITLE
CSI-1109_license

### PR DESCRIPTION
**Pull request by igorgonibm**

### CODE REVIEW

File: NOTICES [hunk 1]
```diff
@@ -1117,6 +1117,7 @@ APACHE 2.0 LICENSED CODE:  The Program includes all or portions of
 the following software which IBM obtained under the terms and conditions
 of the Apache License Version 2.0:
 
+apiextensions-apiserver version 1.1.14
```
Comment (oriyarde) [inline NOTICES]: is this new?

File: NOTICES [hunk 2]
```diff
@@ -1063,24 +1063,24 @@ Licenses and Notices
...
+under the GNU Lesser General Public License 3.0.
 ===========================================================================
 
     Prometheus version 1.8.2
 
-Source code to any of the above-listed packages distributed with Operator
-for IBM Block Storage CSI Driver Version 1.0.0 Third Party Licenses and
-Notices is available at the website below, when a URL is provided, or by
-sending a request to the following address or email: 
+Source code to any of the above-listed packages distributed with IBM
```
Comment (27149chen) [inline NOTICES]: Why the product name is changed?

File: NOTICES [hunk 3]
```diff
@@ -1549,175 +1643,1095 @@ Version 2.0, in this case for the prometheus version 1.8.2 distribution
 The Prometheus systems and service monitoring server
 Copyright 2012-2015 The Prometheus Authors
 
-This product includes software developed at
-SoundCloud Ltd. (http://soundcloud.com/).
+This product includes software developed at
+SoundCloud Ltd. (http://soundcloud.com/).
```
Comment (27149chen) [inline NOTICES]: why there are so many adds?

**Comment by oriyarde** [comment]
@igorgonibm  could we delete this branch?

**Pull request merged**